packages/popover/src/Arrow.js

```diff
@@ -53,9 +52,10 @@ const ArrowAlignment = () =>
     }
   `
 
-const ArrowPlacement = ({ background, borderColor }) => {
-  const bgColor = theme.colors[background]
-  const brColor = theme.colors[borderColor]
+const ArrowPlacement = ({ color, borderColor, ...props }) => {
+  let backgroundColorName = color.indexOf('.') !== -1 ? color : `${color}.base`
```

Comment:
so this is to prevent somebody from doing `error.dark`?

Reply:
never mind, I got my ternary backwards 🤦

Reply:
No, it's when something like `error` is passed as a color. The `getPaletteColor` usage in here isn't able to grab the corresponding background color from the palette to match the `Box` background color unless it's filled in manually here.

Reply:
I can flip the condition around if it makes it easier to read though!